jest.config.js

```diff
@@ -23,7 +23,6 @@ export default {
   ],
   moduleNameMapper: {
     '^react-native$': 'react-native-web',
-    '^@mui/(.*)$': '<rootDir>/node_modules/@mui/$1',
```

Comment:
Is there any particular reason for removing the mui mapper here?

Reply:
The reason the tests were failing was this mapper. This is because mui changed there file structure which makes the mapper invalid. The latest version doesn't require the mui mapper anymore.